src/test/resources/features/dcu/DCUDataInput.feature

```diff
@@ -90,55 +116,79 @@ Feature: DCU Data Input
+    When I create a "MIN" case and move it to the "Data Input" stage
+    And I load and claim the current case
     And I click the "Continue" button on the "IS THE CORRESPONDENT AN MP" page
     Then an error message should be displayed as I must select a correspondent type on this screen
 
   @Validation
   Scenario: User must select an MP from drop down box at Data Input stage
+    When I create a "MIN" case and move it to the "Data Input" stage
+    And I load and claim the current case
     And I click the "Add" button on the "ADD MEMBER OF PARLIAMENT" page
     Then an error message should be displayed as I must select a member of parliament from the drop down
 
   @Validation
   Scenario: User must select a correspondent type from the drop down if the correspondent is not an MP
+    When I create a "MIN" case and move it to the "Data Input" stage
+    And I load and claim the current case
     And I click the "Add" button on the "RECORD CORRESPONDENT DETAILS" page
     Then an error message should be displayed as I have not selected the correspondent type
 
   @Validation
   Scenario: User must enter text in correspondent's Full Name field
+    When I create a "MIN" case and move it to the "Data Input" stage
+    And I load and claim the current case
     And I click the "Add" button on the "RECORD CORRESPONDENT DETAILS" page
     Then an error message should be displayed as I have not entered text in the full name field
 
   @Validation
   Scenario: User must enter text in the text box when creating a Case note at the Data Input stage
+    When I create a "MIN" case and move it to the "Data Input" stage
+    And I load and claim the current case
     And I click the add button when creating a case note
     Then an error message should be displayed as I have not entered text in the Case Note text box
 
   @OtherTests
   Scenario: User removes a correspondent
-    When I fill all mandatory fields on the "Data Input" page with valid data
+    When I create a "MIN" case and move it to the "Data Input" stage
+    And I load and claim the current case
+    And I fill all mandatory fields on the "Data Input" page with valid data
     And I click the "Continue" button
     And I add "Nicola Sturgeon" MP as a correspondent
     And I remove the primary correspondent
     Then there shouldn't be a primary correspondent displayed
 
   @OtherTests
   Scenario: User edits an existing correspondent
-    When I fill all mandatory fields on the "Data Input" page with valid data
+    When I create a "MIN" case and move it to the "Data Input" stage
+    And I load and claim the current case
+    And I fill all mandatory fields on the "Data Input" page with valid data
     And I click the "Continue" button
     And I add "Nicola Sturgeon" MP as a correspondent
     And I edit the primary correspondents name
     Then the correspondents name should be updated
 
```

Comment:
Does the below scenario not overlap with the deadlines feature? If it doesnt than I think we should consider refactoring this scenario into Deadline.

Reply:
Found duplicate scenario in Deadlines feature, removed the DCUDataInput instance of scenario